File: Selenium/Project/HRM_Activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("http://alchemy.hguy.co/orangehrm")
    print("Title: ",driver.title)
    if driver.title == "OrangeHRM":
        print("Title of the Website matches")
    else:
        print("Title doesn't match")
    driver.quit()

File: Selenium/Project/HRM_Activity1.java
package Project;


import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.Test;

public class HRM_Activity1 {

    @Test
    public void testmethod()
    {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver=new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        String title=driver.getTitle();
        Assert.assertEquals(title,"OrangeHRM");
        driver.close();
    }

}
